js/README.md

```diff
@@ -203,6 +197,28 @@ The JS community is a diverse group with a varied list of target environments an
 
 If you think we missed a compilation target and it's a blocker for adoption, please open an issue.
 
+# Community
+
+Full list of broader Apache Arrow [committers](https://arrow.apache.org/committers/) and [projects & organizations](https://github.com/apache/arrow/blob/master/site/powered_by.md).
+
+## Committers
+
+* Brian Hulette, CCRI
+* Paul Taylor, Graphistry, Inc.
+
+## Open Source Projects
+
+* [Apache Arrow](https://arrow.apache.org) -- Parent project for Powering Columnar In-Memory Analytics, including affiliated open source projects
+* [rxjs-mapd](https://github.com/graphistry/rxjs-mapd) -- A MapD Core node-driver that returns query results as Arrow columns
+
+
+
+## Companies & Organizations
+
+* [CCRi](http://www.ccri.com/) -- Commonwealth Computer Research Inc, or CCRi, is a Central Virginia based data science and software engineering company
```

Comment:
This looks good :+1: thanks!

Reply:
@lmeyerov Less hype, more informative, works for me; thanks!